xcpretty.go

```diff
@@ -2,40 +2,57 @@ package main
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/bitrise-io/go-utils/log"
 	"github.com/bitrise-io/go-xcode/xcpretty"
 	version "github.com/hashicorp/go-version"
 )
 
+const (
+	installationCheckError = "failed to check if xcpretty is installed"
+	installError           = "failed to install xcpretty"
+	determineVersionError  = "failed to determine xcpretty version"
+)
+
 // InstallXcpretty installs and gets xcpretty version
 func InstallXcpretty() (*version.Version, error) {
 	fmt.Println()
 	log.Infof("Checking if output tool (xcpretty) is installed")
 
 	installed, err := xcpretty.IsInstalled()
 	if err != nil {
-		return nil, fmt.Errorf("failed to check if xcpretty is installed, error: %s", err)
+		return nil, createError(installationCheckError, err)
```

Comment:
I would argue against using the `createError` method. It increases complexity as:
- it's wrapping a common usage into a function
- we are not using this pattern anywhere else.

An other thing is, please drop the usage of `blah, error: message` and use `blah, message`.

Reply:
I agree with your points, I will inline the errors instead. 👍 

I got the spark to check the official documentation about nested errors. I found [this](https://godoc.org/github.com/pkg/errors#Wrap). It suggests using the `blah: message` format. Whdt?

Reply:
Yeah, that is what I wanted to write 👍